src/htmltojsx.js

```diff
@@ -562,12 +579,9 @@ StyleParser.prototype = {
    */
   toJSXString: function() {
     var output = [];
-    for (var key in this.styles) {
-      if (!this.styles.hasOwnProperty(key)) {
-        continue;
-      }
-      output.push(this.toJSXKey(key) + ': ' + this.toJSXValue(this.styles[key]));
-    }
+    eachObj(this.styles, function(key, value) {
```

Comment:
It's totally fine, don't worry @tsuyoshiwada 😄  I was just wondering about why we couldn't just continue to use a regular `for` loop here.